transmogrifier/sources/xml/datacite.py

```diff
@@ -255,6 +255,15 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
                 source_record_id,
             )
 
+        # publishers
```

Comment:
I considered but decided not to start breaking this out as a field method in the older transforms, I think that work will benefit from a comprehensive rather than piecemeal approach